.github/workflows/benchmark.yml

```diff
@@ -67,6 +67,6 @@ jobs:
           # Show alert with commit comment on detecting possible performance regression
           alert-threshold: '200%'
           comment-on-alert: true
-          fail-on-alert: true
+          fail-on-alert: false
```

Comment:
With `fail-on-alert` set to `false`, will we still get notified with regression, or we need to visit the benchmark page to get the result?

Reply:
The workflow will not be marked as failed but still there will be some comment on the commits as we have now.